src/test/java/org/folio/testutil/SimpleSamlPhpContainer.java

```diff
@@ -0,0 +1,82 @@
+package org.folio.testutil;
+
+import java.io.IOException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.MountableFile;
+
+public class SimpleSamlPhpContainer<C extends SimpleSamlPhpContainer<C>>
+    extends GenericContainer<C> {
+
+  private static final boolean DEBUG = false;
+  private static final Logger logger = LoggerFactory.getLogger(SimpleSamlPhpContainer.class);
+
+  private String baseUrl;
+
+  /**
+   * @param callback either "callback" or "callback-with-expiry"
+   */
+  public SimpleSamlPhpContainer(String okapiUrl, String callback) {
+    super(DockerImageName.parse("kenchan0130/simplesamlphp:1.19.9"));
+    copyResource("server.key", "/var/www/simplesamlphp/cert/server.key");
+    copyResource("server.crt", "/var/www/simplesamlphp/cert/server.crt");
```

Comment:
Don't think these two copies are needed since they are already in the container.